pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1580,6 +1680,46 @@ func (ctrl *PersistentVolumeController) provisionClaimOperation(claim *v1.Persis
 	return pluginName, nil
 }
 
+// provisionClaimOperationCSI provisions a volume using external provisioner async-ly
+// This method will be running in a standalone go-routine scheduled in "provisionClaim"
+func (ctrl *PersistentVolumeController) provisionClaimOperationCSI(
+	claim *v1.PersistentVolumeClaim,
+	plugin vol.ProvisionableVolumePlugin,
+	storageClass *storage.StorageClass) (string, error) {
+	claimClass := v1helper.GetPersistentVolumeClaimClass(claim)
+	klog.V(4).Infof("provisionClaimOperationCSI [%s] started, class: %q", claimToClaimKey(claim), claimClass)
+	// pluginName is not set here to align with existing behavior
```

Comment:
This comment block does not appear to be relevant in this refactored version of the code. You can remove this as `pluginName` is now gone.

Reply:
yep, it should actually be provisionerName instead.

Reply:
fixed in the new commit